aya/src/obj/mod.rs

```diff
@@ -43,11 +43,40 @@ pub struct Object {
     pub(crate) symbols_by_index: HashMap<usize, Symbol>,
 }
 
+#[derive(Debug, Clone, PartialEq)]
+pub enum MapKind {
+    Default,
+    Bss,
+    Data,
+    Rodata,
+}
+
+impl Default for MapKind {
```

Comment:
Is anything using this? If not I'd remove it

Reply:
removed